js/animation_about-us.js

```diff
@@ -0,0 +1,21 @@
+var $animation_elements = $('.about-us__team-members');
```

Comment:
Zmienne globalne się tu panoszą :)

Opakujcie cały kod w tym pliku za pomocą 

```javascript
$(function () {

})
```
(to jest skrót od `$(document).ready(function () {......})`, który dodatkowo robi Wam zamknięty scope na zmienne - mniej zmiennych globalnych - szczęśliwszy programista)

Reply:
https://tenor.com/view/yessir-salute-schoolofrock-ready-lets-gif-5159062

Reply:
chyba nie trzeba dodawać nowej klasy - możesz użyc  `about-us__team-members`

Reply:
Ja bym w ogóle rozważył dodanie specjalnych klas z prefiksami np. `js--about_us`, przeznaczonych do wyszukiwania elementów z poziomu JS. Nie róbcie dla nich osobnych CSSów - chodzi o to, żeby zmiana w CSS (np. refaktoring nazw klas) nie sprawiła, że część skryptów przestanie działać. Ale to tylko propozycja